index.d.ts

```diff
@@ -145,18 +145,23 @@ declare module '@primer/components' {
 
   export const StyledOcticon: React.FunctionComponent<StyledOcticonProps>
 
-  export interface DropdownProps extends React.Props<any>, StyledSystem.ColorProps, StyledSystem.SpaceProps, ButtonProps {
-    as?: React.ReactType
-    title?: string | React.ReactNode
-  }
+  export interface DropdownProps extends CommonProps, Omit<React.HTMLAttributes, 'color'> {}
+
+  export interface DropdownItem extends CommonProps, Omit<React.HTMLAttributes<HTMLLIElement>, 'color'> {}
 
   export interface DropdownMenuProps extends CommonProps, Omit<React.HTMLAttributes<HTMLUListElement>, 'color'> {
     direction?: string
   }
 
+  export interface DropdownButtonProps extends ButtonProps, Omit<React.HTMLAttributes, 'color'> {}
```

Comment:
@T-Hugs I noticed that there is no `HTMLSummaryElement` available, so i just left this as `React.HTMLAttributes` is that the correct thing to do?

Reply:
Looks like `<summary>` itself takes `React.HTMLAttributes<HTMLElement>` so I'd use that